lib/ansible/modules/windows/win_scheduled_task.ps1

```diff
@@ -25,6 +25,8 @@ $ErrorActionPreference = "Stop"
 $params = Parse-Args $args;
 
 $days_of_week = Get-AnsibleParam $params -name "days_of_week"
+$interval = Get-Attr $params -name "interval" -default $null
+$interval_unit = Get-Attr $params -name "interval_unit" -default minutes
```

Comment:
interval unit can only be one out of a list. You can do this:
```powershell
$interval_unit = Get-AnsibleParam -obj $params -name "interval_unit" -type "bool" -default "minutes" -validateset "days","hours","minutes","seconds"
```

And then you don't have to check below anymore if it is one of these options.